.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
Shouldn’t it be up to each developer to choose which version they want to develop with?

Reply:
I think it would be good to standardize (maybe using something like [asdf](https://asdf-vm.com/#/)) so that everyone has the exact same versions of Node/Python/whatever other languages we use.

Reply:
I removed it, at least for now, to unblock the PR. We can re-add later if it makes sense.